packages/action-menu/src/ActionMenu.ts

```diff
@@ -87,6 +87,7 @@ export class ActionMenu extends ObserveSlotPresence(
     protected override render(): TemplateResult {
         return html`
             <sp-action-button
+                aria-describedby=${this.description}
```

Comment:
You don't seem to be making this same associating in the `Picker.ts` code...

Reply:
done!